.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "[javascript]": {
+    "editor.formatOnSave": false
+  }
+}
```

Comment:
I had to make this change because my editor kept messing up the formatting of files in the project. I can add `settings.json` to out `.gitignore` file so this change is not tracked by git. Is that ok?

Reply:
Yes @rotimi-babalola. Please add it to `.gitignore`